logger.js

```diff
@@ -59,11 +59,11 @@ module.exports = function(config) {
 	};
 
 	function snipFragileData(data) {
-		for (const key in data) {
+		Object.keys(data).forEach(key => {
```

Comment:
why not `return Object.keys(data).map(key => {`?

Reply:
Because it will return an array instead of an object.